62_CAD/DrawResourcesFiller.cpp

```diff
@@ -897,11 +907,28 @@ void DrawResourcesFiller::addGeoreferencedImage(image_id imageID, const Georefer
 		return;
 	}
 
+	// Query imageType
+	auto cachedImageRecord = imagesCache->peek(imageID);
+	if (!cachedImageRecord)
+	{
+		m_logger.log("addGeoreferencedImage was not called immediately after enforceGeoreferencedImageAvailability!", nbl::system::ILogger::ELL_ERROR);
+		assert(false);
+	}
+
+	// Generate upload data
+	auto uploadData = generateTileUploadData(cachedImageRecord->type, NDCToWorld, cachedImageRecord->georeferencedImageState.get());
```

Comment:
It seems like it's caculating tiles to load and then requesting loading them.
maybe a better name along "computeTilesAndLoad"
I'm also thinking about if separating this into two function "compute" + "load" would do us any benefit when we use futures (instead of actually loading them on demand)
but I don't think so, this function can just return a bunch of futures to hold onto and wait upon...